src/Pages/ReviewWrite/ReviewWrite.js

```diff
@@ -1,10 +1,10 @@
 import React, { useEffect, useState } from 'react';
-import { useParams } from 'react-router-dom';
-import ReviewList from '../../Components/ReviewList/ReviewList';
+import { Link, Navigate, useNavigate, useParams } from 'react-router-dom';
 import Melon from './Melon';
 import './ReviewWrite.scss';
 
 const ReviewWrite = () => {
+  const Navigate = useNavigate();
```

Comment:
```suggestion
  const navigate = useNavigate();
```
변수명은 파스칼 케이스가 아닌 카멜 케이스로 작성해주세요